packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -266,6 +267,14 @@ export default function DataviewsTemplates() {
 					filteredTemplates = filteredTemplates.filter( ( item ) => {
 						return item.author_text === filter.value;
 					} );
+				} else if (
```

Comment:
We could though simplify the conditions here, but it's fine for now that we are iterating so much..